portfolio/src/main/webapp/script.js

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 /**
- * Adds the next meme in the array to the page.
+ * Functions for all the buttons
```

Comment:
Instead of having one doc comment to cover all the functions, there should be a separate comment on each function describing why someone would call it. E.g. for this one it might say:

```
/**
 * Displays the next available meme.
 */
```

or something like that.